cmd/libs/go2idl/unstructured-conversion-gen/generators/unstructured-conversion.go

```diff
@@ -0,0 +1,350 @@
+/*
```

Comment:
@smarterclayton - can you please take a look if that makes any sense to you? It's not yet finished, but before I spent too much time on it, I would like to get your opinion about it.